client/api/volume/v1beta3/api.proto

```diff
@@ -5,128 +5,129 @@ package v1beta3;
 option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1beta3";
 
 service Volume {
-    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for
-    // all volumes on a Disk device
+    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for all volumes on a Disk device.
```

Comment:
maybe ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for all volumes from a given disk number and partition number (optional)

Reply:
fixed